tests/test_errors.py

```diff
@@ -21,3 +24,19 @@ def foo():
 """)
     with pytest.raises(ValueError):
         v.get_unused_code(min_confidence=150)
+
+
+def test_non_utf8_encoding(v):
+    code = """\
+def foo():
+    pass
+
+foo()
+"""
+    tests = os.path.join(REPO, 'tests')
+    with tempfile.NamedTemporaryFile(mode='wb', dir=tests, suffix='.py') as f:
```

Comment:
Can we just remove the `dir` parameter and the `tests` variable and let Python choose a temporary dir?

Reply:
Would it still be a good idea to leave `DIR`, `REPO` and `WHITELISTS` in `__init__.py`?

Reply:
> Can we just remove the dir parameter and the tests variable and let Python choose a temporary dir?

I'm trying to remember why I did that on the firsthand? 😅

Reply:
You can leave DIR, REPO and WHITELISTS in __init__.py